style.css

```diff
@@ -676,6 +748,82 @@ a {
     grid-row: 2/3;
   }
 
+  /* popup project details */
+  .desk {
+    border: 1px solid red;
+  }
+
+  .pop-up-project-section {
+    background-color: rgba(0, 0, 0, 0.5);
+    padding: 50px;
+    position: fixed;
+    top: 0;
+    left: 0;
+    z-index: 2000;
+    display: none;
+    height: auto;
+  }
+
+  .popup {
+    display: block;
+  }
```

Comment:
- [ ] I must commend that you have made a great effort working on your pop-up modal, but I notice some styling misalignment and this shows a high difference between your design and the Figma design you worked on. Below is some highlights of where I would suggest you pay attention to:

  - Please make the image and the text in the pop-up modal aligned as shown in the Figma design
  - The `See live` and `See Source` buttons are far away from each other and this shows a great difference from the Figma design
  - I notice when the pop-up modal is displayed they are not well positioned, in such a way that when I scroll down the main page underneath shows, which is not supposed to be like that. Kindly give it the property of `backdrop filter` with a value of `blur(8px)`
  - Please make your `image` and the `buttons` align to the bottom.

Your Design 👇 

![aub-pp1](https://user-images.githubusercontent.com/78634841/209224536-e2d9c07e-e390-4a25-acb8-a49c7a3ce22e.png)

Figma Design 👇 

![pp2-real](https://user-images.githubusercontent.com/78634841/209224694-243466ce-3aa8-4728-a8e8-243e5290ef41.png)

 > Kindly stick to the Figma design as much as possible in order to show some consistency in your work and the ability to replicate any design given to you by a client.